### ISSUE TITLE
Ctrl + C ？？？

### ISSUE THREAD

**Issue by MIMAZHAN** [xz-zone/Webpackfind#1]
https://github.com/Threezh1/JSFinder/blob/master/JSFinder.py

**Comment by xz-zone** [comment]
No, I just added my own method based on the jsfinder.py source. Read and traverse webpack compressed JS.
